src/analytics/pageViewTracker/pageViewTracker.ts

```diff
@@ -33,8 +43,10 @@ export class PageViewTracker extends BaseTracker {
     }
 
     //trigger form error tracking
+
     const errorTrigger = document.getElementsByClassName("govuk-error-message");
-    if (errorTrigger.length) {
+
+    if (errorTrigger.length && this.enableFormErrorTracking) {
```

Comment:
Yes, I believe so. The feature flags are set up to allow toggling of the tracker functionality. This means that even if there is an error message and we don't want to track form errors, the page view tracker should still execute (provided it has been enabled).

Reply:
Sorry just added a comment below because I wanted to double check with Sam what he would be expecting. His feedback was that we don't want to fall-back to the pageViewTracker if the formErrorTracker is turned off, so I think we need to slightly amend the logic here. Something along the lines of:
```javascript
if (errorTrigger.length) {
  if (this.enableFormErrorTracking) {
    ...do form error tracking logic
  }
  /*
   * Keeping this outside the `enableFormErrorTracking` check means that we return
   * early and avoid the PageViewTracker, regardless of whether the formErrorTracking
   * is on or not.
   */
  return false;
}
```